lib/reruby/git_autocommit.rb

```diff
@@ -0,0 +1,25 @@
+require 'git'
+
+module Reruby
+  class GitAutocommit
+    AUTOCOMMIT_MSG = 'Reruby autocommit before refactoring'.freeze
```

Comment:
Not a big deal, and could be done later on, but I think this could be another config option (--autocommit-message ? ) with this text as default (in the config itself)